cypress/integration/models/stakeholdergroups.ts

```diff
@@ -15,11 +27,15 @@ export class Stakeholdergroups {
     }
 
     protected fillName(name: string): void {
-        inputText(stakeholdergroupName, name);
+        inputText(stakeholdergroupNameInput, name);
     }
 
     protected fillDescription(description: string): void {
-        inputText(stakeholdergroupDescription, description);
+        inputText(stakeholdergroupDescriptionInput, description);
+    }
+
+    protected selectMember(member: string): void {
```

Comment:
This abstraction is not needed, we can simply call the selectMember in utils directly. Also, using same name for both the functions is confusing.

Reply:
@nitishSr IMO, `utils` file is containing widget methods where we are using base features of cypress framework to contact widgets on UI. 
- I have renamed `selectMember` to `select` in utils file. We can use this in common for all the tests like stakeholder has dropdown for job function, Bussiness service has dropdown etc. 
- If you want to apply simply calls to utils methods then it should be applied to all the other methods. 
- Please let me know your views